Dockerfile

```
@@ -0,0 +1,18 @@
+FROM node:16.16.0-alpine AS builder
+
+WORKDIR /app
+COPY package*.json ./
+RUN npm ci
+COPY tsconfig*.json ./
+COPY src src
+RUN npm run build
```

You can simplify this.

```suggestion
WORKDIR /app
COPY . .
RUN npm ci \
    npm run build
RUN ```